docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+DALI GPU implementations are written in CUDA. However there are open source community efforts
+that are enabling running CUDA-based applications on other GPU architectures, but DALI
+doesn't officially support it.
+
+Q: When to use DALI and when RAPIDS?
+####################################
+A: RAPIDS is better suited for general-purpose machine learning and data analytics.
+DALI is a specialized tool for Deep Learning workflows, and it's aimed to accelerate dense data
+processing and overlap it with the network execution.
+
+Q: Is Triton + DALI still significantly better than preprocessing on CPU, when minimum latency i.e. batch-size = 1 is desired?
+##############################################################################################################################
+A: That can depend on the particular case and what other tool we are comparing to.
+One of DALI's advantages is the fact that it can optimize batch processing,
+but even for batch-size = 1, it might be worth using DALI because of the GPU implementation.
+
+Q: Are there any examples of using DALI for volumetric data?
+############################################################
+A: Yes, e.g DALI was used to achieve high performance in NVIDIA’s MLPerf submission for UNet3D.
+You can read an interesting article about it `here <https://developer.nvidia.com/blog/accelerating-medical-image-processing-with-dali>`_.
+You can see the DALI pipeline that was used `in this example <https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/Segmentation/nnUNet/data_loading/dali_loader.py>`_.
+
+Q: Where can I find more details on using the image decoder and doing image processing?
+#######################################################################################
+A: You can always refer to `the relevant section of the DALI documentation <../examples>`_
+where you can find multiple examples of DALI used in different use-cases. For the start,
+you can also watch `our introductory talk on this GTC <https://www.nvidia.com/gtc/session-catalog/#/session/1636566824182001pODM>`_.
+
+Q: Does DALI utilize any special NVIDIA GPU functionalities?
+############################################################
+A: Yes, DALI uses `NVJPEG <https://developer.nvidia.com/blog/loading-data-fast-with-dali-and-new-jpeg-decoder-in-a100/>`_ -
+special HW unit offloading JPEG image decoding, `NVDEC <https://developer.nvidia.com/nvidia-video-codec-sdk>`_ -
+HW video decoder, `GPUDirect Storage <https://developer.nvidia.com/gpudirect-storage>`_ -
+the ability to load data directly to the GPU to avoid a slow round trip through CPU.
+
+Q: Can DALI operate without GPU?
+################################
+A: Yes, DALI can operate without GPU but it is not optimized for the CPU itself.
+The main goal of this functionality is to enable the development of the DALI pipeline on
+machines where GPU is not available (like laptops), with an ability to later deploy the DALI
+pipeline on a GPU-capable cluster.
```

Comment:
```suggestion
A: Not 100%. Most of operators have CPU and GPU variants and a pipeline where all operators are
run on CPU doesn't require a GPU to run. However, DALI is predominantly a GPU library and CPU
operators are not as thouroughly optimized.
The main goal of this functionality is to enable the development of the DALI pipeline on
machines where GPU is not available (like laptops), with an ability to later deploy the DALI
pipeline on a GPU-capable cluster.
```

Reply:
The experimental video reader supports CPU, bit its functionality is very rudimentary.